workflows/offline_ml_diags/tests/test_compute_diags.py

```diff
@@ -53,7 +53,7 @@ def data_path(tmpdir):
     batch_kwargs=batch_kwargs,
     scaler_type="standard",
     scaler_kwargs={},
-    additional_variables=None,
+    additional_variables=[],
```

Comment:
This was intentional. Since I am using dataclasses, it is not possible (or desirable) to do things like this:
```
self.additional_variables: List[str] = additional_variables or []
```
This is a small backwards incompatible change,  but will only appear if someone has a `additional_variables: null` in their config.

Reply:
> will only appear if someone has an additional_variables: null in their config yaml (which I don't think has ever been done).

OK, in that case just wanted to check on this point.